app/root/components/Document/Document.tsx

```diff
@@ -49,7 +48,6 @@ const Document = ({ children, title }: { children: React.ReactNode; title?: stri
       <body>
         <RootProviders>{children}</RootProviders>
         <ScrollRestoration />
-        <LiveReload />
```

Comment:
does Vite handle this somewhere else? why dont we want this?

Reply:
Yup, it's no longer needed, `<Scripts />` does this job now. 

https://remix.run/docs/en/main/future/vite#hmr--hdr